src/persist/src/operators/input.rs

```diff
@@ -105,6 +103,7 @@ impl<K: TimelyData> PersistentUnorderedHandle<K> {
 }
 
 /// A persistent equivalent of [UnorderedHandle::session]'s return type.
+#[allow(missing_debug_implementations)]
```

Comment:
nit: did you want to keep that as-is before merging? no strong preference from me just curious

Reply:
yeah. i played with this for a bit but didn't come up with a fix I liked. plus I'd like to just delete this whole module, there's no use for it in mz